src/Colors.ts

```diff
@@ -19,8 +19,44 @@ interface RGBColor {
   r: number;
   g: number;
   b: number;
+  a?: number;
 }
 
+type HEXColor = string;
```

Comment:
I'd be ok to add a dependency with this package instead: https://www.npmjs.com/package/color?

Reply:
sure, will take a look at it later
thanks!